src/components/SearchField.tsx

```diff
@@ -53,6 +53,10 @@ const SearchField: React.FC = () => {
 
   const [genreToSearch, setGenreToSearch] = useState<string>('none');
 
+  const [isYearIncrease, setIsYearIncrease] = useState<boolean>(false);
+
+  const [isYearDecrease, setIsYearDecrease] = useState<boolean>(false);
```

Comment:
Here, I thought I could create separate states for fromYear and toYear in order to change these states differently depending on what the user uses(clicks) and show a different message accordingly. State changes for each data range (fromYear and toYear) hence I decided to use separate states to avoid overlapping.

Reply:
Thanks for a hint. I have created one state for both date ranges and set a massage as "Date is invalid" for both of them. I decided to create two different states in the first place? because I had in mind an idea to show different message